diff --git a/node_modules/@narative/gatsby-theme-novela/src/sections/article/Article.Share.tsx b/node_modules/@narative/gatsby-theme-novela/src/sections/article/Article.Share.tsx
index d006918..7d22f30 100644
--- a/node_modules/@narative/gatsby-theme-novela/src/sections/article/Article.Share.tsx
+++ b/node_modules/@narative/gatsby-theme-novela/src/sections/article/Article.Share.tsx
@@ -26,7 +26,7 @@ interface MenuFloatState {
 const MENU_WIDTH: number = 225;
 const MENU_HEIGHT: number = 46;
 
-function ArticelShare() {
+function ArticleShare() {
   const [colorMode] = useColorMode();
   const [text, setText] = useState("");
   const [focus, setFocus] = useState(false);
@@ -124,7 +124,7 @@ function ArticelShare() {
   }, [show]);
 
   /**
-   * Small workaround to set the focus once the x and y positiosn are set.
+   * Small workaround to set the focus once the x and y positions are set.
    * If this is not here the user would see a quick flash of the floating bar
    * in its old position and then animating to the new location. We don't want that.
    */
@@ -183,7 +183,7 @@ function ArticelShare() {
   );
 }
 
-export default ArticelShare;
+export default ArticleShare;
 
 function ReferralLink({ disabled, share, children }) {
   function handleClick(event) {
